docs/patterns.rst

```diff
@@ -114,6 +118,86 @@ a ``categorical`` slot.
 
 Rasa automatically represents (featurises) this as a one-hot encoding of the values: ``(1,0,0)``, ``(0,1,0)``, or ``(0,0,1)``.
 
+**Slot features**
+
+.. _slot_features:
+
+When Rasa Core runs training against your story, the presence of a ``Slot`` entry will be used to help 
+determine the next action that should be taken. 
+
+This works best with ``CategoricalSlot`` slot types. A ``TextSlot`` can have any value, but it only has one 
+feature - set ``(1)`` or not set ``(0)``. A ``CategoricalSlot`` has a set number of values and a feature for each. Rasa core 
+will be able to make decisions based not only on whether the value is set but also on the value itself.
+
+.. code-block:: yaml
+
+    restaurant_availability:
+        type: categorical
+        values:
+        - unknown
+        - booked-out
+        - waiting-list
+        - available
+
+When the ``restaurant_availability`` slot is set Rasa Core will be able to determine if the restaurant in question is
+available and choose radically different actions to perform based on the value.
+
+The ``Slot`` *might* be set by Rasa Core itself from entities detected by the NLU module, but usually you would 
```

Comment:
let's not say *might*, but rather be explicit. If NLU detects this as an entity, and there is a slot with the same name, this value will be set as a slot.

Reply:
agreed